src/main/java/mate/academy/spring/validation/PasswordValidator.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.spring.validation;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class PasswordValidator implements ConstraintValidator<Password, Object> {
```

Comment:
```suggestion
public class PasswordValidator implements ConstraintValidator<Password, UserRequestDto> {
```
`isValid()` should be overridden properly with that
next time check out the documentation of what are parameters in `<>` represent